_episodes/06-files.md

```diff
@@ -85,7 +85,9 @@ inflammation-01.csv
 ~~~
 {: .output}
 
-![Analysis of inflammation-01.csv](../fig/03-loop_49_1.png)
+![Three plots are generated by the code applied to inflammation-01.csv. 
+The centre plot with the y-label max is triangular.  
+The other two plots are jagged triangles.](../fig/03-loop_49_1.png)
```

Comment:
Since these plots are the same as the those in the three previous images, but side-by-side here, I wonder if it makes sense to skip re-describing the physical characteristics of the lines and say something like:  "Three side-by-side subplots show, from left to right, the average, maximum, and minimum values per day for the inflammation-01.csv data".